src/Entity/ApiToken.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ApiTokenRepository;
+use DateTimeImmutable;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: ApiTokenRepository::class)]
+#[ORM\Table(name: "user_api")]
+class ApiToken
+{
+    #[ORM\Id]
+    #[ORM\Column(type: 'string', length: 50)]
+    private $userId;
+
+    #[ORM\Column(name: 'key', type: 'string', length: 50)]
+    private $apiToken;
+
+    #[ORM\Column(name: 'expiration_date', type: 'datetime_immutable', nullable: true)]
+    private $expiresAt;
+
+    public function getUserId(): ?string
```

Comment:
Why is is nullable if the user_id field is not?

Reply:
My bad :)